### PR TITLE
Initial pass at displaying contextual related records

### PR DESCRIPTION
**Pull request by willvincent**
Working except for display of relation data. Queries get properly generated and run on click of the foreign key icon (should probably add a style to make the mouse a 'pointer' on those), just console logs the result currently so needs a dialog to display result, otherwise it should more or less be good to go :+1:

### CODE REVIEW

**Comment by Paxa** [comment]
Just checked, it works well.
I'm going to change `float: right` for icon, I like it more at the end of a text, but not sure yet...

Still thinking how to present that data: show some dialog or tooltip with preview and button to go to that table. Or just change current table and apply filtering (like sequel pro does, but then it's hard to go back to first table)

File: app/models/table.js [hunk 1]
```diff
@@ -545,6 +545,23 @@ class Table extends ModelBase {
...
+          kcu.column_name,
+          ccu.table_name AS foreign_table_name,
+          ccu.column_name AS foreign_column_name
+      FROM
+          information_schema.table_constraints AS tc
+          JOIN information_schema.key_column_usage 
+              AS kcu ON tc.constraint_name = kcu.constraint_name
+          JOIN information_schema.constraint_column_usage 
+              AS ccu ON ccu.constraint_name = tc.constraint_name
+      WHERE tc.table_name = '${this.table}' AND constraint_type = 'FOREIGN KEY'
```
Comment (Paxa) [inline app/models/table.js]: Need to add `constraint_schema = '${this.schema}'`, will make it

**Comment by willvincent** [comment]
> Still thinking how to present that data: show some dialog or tooltip with preview and button to go to that table. Or just change current table and apply filtering (like sequel pro does, but then it's hard to go back to first table)

I think a dialog or tooltip makes the most sense here, maybe with a button to open that table scrolled to or filtered to that result.

**Comment by Paxa** [comment]
screen shot 2019-02-10 at 17 18 23
Made it like this for now
Would be nice if you can how it works https://postbird.paxa.kuber.host/2019_02_10.09_22-master-5209ffe

**Pull request merged**